src/components/timeline/_macro.njk

```diff
@@ -0,0 +1,14 @@
+{% macro onsTimeline(params) %}
+
+    <div class="timeline">
+
+        {% for item in params.items %}
+        <div class="timeline__item">
+            <h2 class="timeline__heading">{{ item.heading }}</h2>
+            {{ caller() if caller else item.content | safe }}
```

Comment:
I don't think (?) `caller()` will work inside the loop in this way. You would need to `call` the timeline component multiple times for each item. If this is the case you would just need to keep `item.content | safe`

Reply:
The `caller()` is mainly useful for including other components inside the content of the main component i.e. a list component instance inside an error component.

Reply:
Thanks Matt. Will update.